Frontend/src/pages/admin/CsvParse.vue

```diff
@@ -0,0 +1,169 @@
+                <q-btn class="q-mt-md" color="primary" label="Check Students" :loading="uploading" @click="parseCsv"/>
+            </div>
+        </div>
+
+        <div id="container" class="row flex justify-center" v-if="submitted">
+            <div class="col">
+                <q-card flat bordered class="q-mb-md">
+                    <q-card-section>
+                        <div class="text-h6 q-mb-md">Students who have not enrolled</div>
+                    </q-card-section>
+                    <div v-if="unenrolledStudents.length > 0" class="q-ml-md">
+                        <q-chip icon="email" size="md" v-for="student in unenrolledStudents" :key="student">
+                            {{ student }}
+                        </q-chip>
+                    </div>
+                    <q-card-actions vertical align="right">
+                        <q-btn @click="showPopup=true" flat>Notify all</q-btn>
+                        <q-btn v-clipboard:copy="unenrolledStudents" v-clipboard:success="onCopy" flat>Copy List</q-btn>
+                    </q-card-actions>
+                </q-card>
+
+                <q-card flat bordered class="q-mb-md">
+                    <q-card-section>
+                        <div class="text-h6 q-mb-md">Students who have not created a new Co-op</div>
+                    </q-card-section>
+                    <div v-if="unregisteredStudents.length > 0" class="q-ml-md">
+                        <q-chip icon="email" size="md" v-for="student in unregisteredStudents" :key="student">
+                            {{ student }}
+                        </q-chip>
+                    </div>
+                    <q-card-actions vertical align="right">
+                        <q-btn v-clipboard:copy="unregisteredStudents" v-clipboard:success="onCopy" flat>Copy List</q-btn>
+                    </q-card-actions>
+                </q-card>
+
+                <q-dialog v-model="showPopup">
+                    <NotificationPopup :title="selectedCourse" :course="specificCourse" :students="unenrolledStudents" />
+                </q-dialog>
+            </div>
+        </div>    
+    </q-page>
+</template>
+
+<script>
+import NotificationPopup from "../../components/admin/NotificationPopup.vue";
+
+export default {
+    name: "CSVParserPage",
+    components: {
+        NotificationPopup,
+    },
+    data: function() {
+        return {
+            selectedYear: "",
+            yearList: [],
+            seasonList: [],
+            selectedSeason: "",
+            courseList: [],
+            selectedCourse: "",
+            unenrolledStudents: [],
+            unregisteredStudents: [],
+            csvFile: null,
+            uploading: false,
+            courseOfferingId: "",
+            showPopup: false,
+            specificCourse: [],
+            submitted: false,
+        };
+    },
+    created: function() {
+        this.$axios.get("/courses").then(resp => {
+            resp.data.forEach(course => {
```

Comment:
If these routes return a list then I think you can just set the variable directly instead of looping over the array, e.g. `this.courseList = resp.data`